README.md

```diff
@@ -2,7 +2,7 @@
 
 Expand snippets matching the current prefix with `tab` in Atom.
 
-To add your own snippets, select the _Atom > Open Your Snippets_ menu option if you're using OSX or the _File > Open Your Snippets_ menu option if you're using Windows or Linux.
+To add your own snippets, select the _Atom > Open Your Snippets_ menu option if you're using OSX, or the _File > Open Your Snippets_ menu option if you're using Windows, use _Edit > Open Your Snippets_ if you are using Linux.
```

Comment:
Ok, so after thinking about this, I think the best flow would go something like this:
> To add your own snippets [...] using OS X, the [...] using Windows, or the _Edit > Open Your Snippets_ menu option if you are using Linux.

Does that sound good to you?

(Also, note the new space in `OS X` - we're starting to standardize the usage of OS X over Mac and OSX, /cc @thomasjo)